aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -118,6 +118,10 @@ Tensor& sub_(Tensor& self, const Tensor& other, Scalar alpha) {
   return native::sub_out(self, self, other, alpha);
 }
 
+Tensor rsub(const Tensor& self, const Tensor& other, Scalar alpha) {
+  return native::sub(other, self, alpha);
```

Comment:
Change this call to `at::sub` (instead of `native::sub`) and remove the derivative definition